src/LanguageServer/Test/CompletionTests.cs

```diff
@@ -838,8 +834,8 @@ class x(object):
 
             result = cs.GetCompletions(analysis, new SourceLocation(3, 22));
             result.Should().HaveLabels("import")
-                .And.NotContainLabels("abc", @"unittest", "abs", "dir")
-                .And.Subject.ApplicableSpan.Should().Be(3, 20, 3, 26);
+                  .And.NotContainLabels("abc", @"unittest", "abs", "dir")
```

Comment:
Another style nit, but a lot of files are getting changed with spacing things. You may want to see what settings are missing to prevent this.